src/api/app/assets/javascripts/webui/sidebar-menu.js

```diff
@@ -0,0 +1,38 @@
+$(document).ready(function () {
+
```

Comment:
This extra line could be removed. We don't really lint JS files right now... so we're not super strict on this.